app/src/main/java/com/mathroule/demoapp/Helper.java

```diff
@@ -0,0 +1,10 @@
+package com.mathroule.demoapp;
+
+import java.util.Random;
+
+public class Helper {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><table>
  <tbody>
    <tr>
      <td><span data-href="https://github.com/mathroule/demo-app/blob/15d3b9565f791bb5c2bdb22540e7aa2ee8420984/app/src/main/java/com/mathroule/demoapp/Helper.java#L5"></span>The utility class name ‘Helper’ doesn’t match ‘[A-Z][a-zA-Z0-9]+(Utils?</td>
      <td>Helper)’</td>
    </tr>
  </tbody>
</table>
</td>
    </tr>
  </tbody>
</table>